src/Commands/DocumentSets/AddDocumentSet.cs

```diff
@@ -19,6 +19,9 @@ public class AddDocumentSet : PnPWebCmdlet
         [ValidateNotNullOrEmpty]
         public string Name;
 
+        [Parameter(Mandatory = false)]
+        public string Folder;
```

Comment:
Can you please modify this to use `FolderPipeBind` ? You can check & copy over the implementation that we have in `Add-PnPFolder` cmdlet, would make things easier and consistent for users

Reply:
Thanks @gautamdsheth I can update this, however I used the logic in `Add-PnPListItem` which will create the folder if it doesn't exist. It also uses the relative path from the `List.RootFolder` e.g. `"Folder A/Folder B"` as in this command we are specifying the List already which makes sense rather than having to repeat the List name/path in the `FolderPipeBand` e.g. `"Shared Documents/Folder A/Folder B"`

Reply:
@danielcecil - will merge it and make some changes later. The advantage of pipebind is that we can easily use it with `Get-PnPFolder` or related cmdlets. If you notice the code, it does create the folder if it doesn't exist in that cmdlet.